.gitignore

```diff
@@ -1,4 +1,5 @@
 .puli/
+.idea
```

Comment:
This should not be in the .gitignore. 

You should put this in your global .gitignore file.